meinberlin/assets/scss/components/_hero_unit.scss

```diff
@@ -0,0 +1,40 @@
+.hero-unit {
+    position: relative;
+    min-height: 25em;
+
+    background: no-repeat center center;
+    background-size: cover;
+
+    &:before {
+        content: ' ';
```

Comment:
In other places (e.g. `_tabs.scss`) we use `""` as a value here. I would prefer to have this consistent.